modules/catalog-compaction/src/integrationTest/java/org/apache/ignite/internal/catalog/compaction/ItNodeRecoveryAfterCatalogTruncatedAboveStoredLwmTest.java

```diff
@@ -0,0 +1,133 @@
+
+import static org.apache.ignite.internal.TestDefaultProfilesNames.DEFAULT_AIMEM_PROFILE_NAME;
+import static org.apache.ignite.internal.TestDefaultProfilesNames.DEFAULT_AIPERSIST_PROFILE_NAME;
+import static org.apache.ignite.internal.TestDefaultProfilesNames.DEFAULT_ROCKSDB_PROFILE_NAME;
+import static org.apache.ignite.internal.TestDefaultProfilesNames.DEFAULT_TEST_PROFILE_NAME;
+import static org.apache.ignite.internal.TestWrappers.unwrapIgniteImpl;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.nio.file.Path;
+import org.apache.ignite.InitParametersBuilder;
+import org.apache.ignite.internal.ClusterPerTestIntegrationTest;
+import org.apache.ignite.internal.app.IgniteServerImpl;
+import org.apache.ignite.internal.catalog.CatalogManager;
+import org.apache.ignite.internal.configuration.IgnitePaths;
+import org.apache.ignite.internal.hlc.HybridTimestamp;
+import org.apache.ignite.internal.lowwatermark.LowWatermarkImpl;
+import org.apache.ignite.internal.vault.VaultEntry;
+import org.apache.ignite.internal.vault.VaultManager;
+import org.apache.ignite.internal.vault.VaultService;
+import org.apache.ignite.internal.vault.persistence.PersistentVaultService;
+import org.junit.jupiter.api.Test;
+
+class ItNodeRecoveryAfterCatalogTruncatedAboveStoredLwmTest extends ClusterPerTestIntegrationTest {
+    private static final String ZONE_NAME = "TEST_ZONE";
+    private static final String TABLE_NAME = "TEST_TABLE";
+
+    @Override
+    protected String getNodeBootstrapConfigTemplate() {
+        return "ignite {\n"
+                + "  network: {\n"
+                + "    port: {},\n"
+                + "    nodeFinder.netClusterNodes: [ {} ]\n"
+                + "  },\n"
+                + "  storage.profiles: {"
+                + "        " + DEFAULT_TEST_PROFILE_NAME + ".engine: test, "
+                + "        " + DEFAULT_AIPERSIST_PROFILE_NAME + ".engine: aipersist, "
+                + "        " + DEFAULT_AIMEM_PROFILE_NAME + ".engine: aimem, "
+                + "        " + DEFAULT_ROCKSDB_PROFILE_NAME + ".engine: rocksdb"
+                + "  },\n"
+                + "  storage.engines: { "
+                + "    aipersist: { checkpoint: { "
+                + "      intervalMillis: " + 250
+                + "    } } "
+                + "  },\n"
+                + "  clientConnector.port: {},\n"
+                + "  rest.port: {},\n"
+                + "  failureHandler.dumpThreadsOnFailure: false\n"
+                + "}";
+    }
+
+    @Override
+    protected void customizeInitParameters(InitParametersBuilder builder) {
+        builder.clusterConfiguration(aggressiveLowWatermarkIncreaseClusterConfig());
+    }
+
+    private static String aggressiveLowWatermarkIncreaseClusterConfig() {
```

Comment:
I think I've seen this config in several places already, should we move it to the parent class?